src/View/Helper/FormHelper.php

```diff
@@ -106,6 +114,13 @@ public function input($fieldName, array $options = [])
                 if (!isset($options['inline'])) {
                     $options['inline'] = $this->checkClasses('checkbox-inline', (array)$options['label'])
                         || $this->checkClasses('radio-inline', (array)$options['label']);
+
+                    if (!$this->_align) {
+                        $this->templates([
+                            'checkboxContainer' => '<div class="checkbox">{{content}}</div>',
```

Comment:
All this template juggling is getting ugly and will be difficult to maintain in future. We should have two config file will all templates and just select the one required whether user wants horizontal or not.